packages/common/src/services/audius-backend/solana.ts

```diff
@@ -197,10 +196,10 @@ export const getUserbankAccountInfo = async (
  * Defaults to AUDIO mint and the current user's wallet.
  */
 export const createUserBankIfNeeded = async (
+  sdk: AudiusSdk,
```

Comment:
Seems like this is taking in a reference to `audiusBackendInstance` just to call `libs.getCurrentUser().wallet`? Can we just remove that and just make the address required?

If needed, can get the 'current user' wallet address from account selectors.

Reply:
oh and we return `res.userBank` at the bottom of this fn so all good!